pkg/health/client/client.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Authors of Cilium
+// Copyright 2018 Authors of Cilium
```

Comment:
Nit: Usually update the copyrights in the files to include the current year but not remove the old year, ie `2017-2018`.